developer-workflow/lang-changes.rst

```diff
@@ -1,37 +1,40 @@
-programming language. This document is meant to explain exactly what changes
-have a reasonable chance of being considered and what the process is to propose
-changes to the language.
+Changing Python
+===============
 
+On occasion people come up with an idea for changing or improving Python.  This
+page explains how to propose changes and what to expect during the process.
 
-What qualifies
+
+Considerations
 --------------
-First and foremost, it must be understood that changes to the Python
-programming language are difficult to make. When the language changes,
-**every** Python programmer already in existence and all Python programmers to
+
+First, understand that changes to Python are difficult to make. When the
+language changes, **every** Python programmer and all Python programmers to
 come will end up eventually learning about the change you want to propose.
 Books will need updating, code will be changed, and a new way to do things will
-need to be learned. Changes to the Python programming language are never taken
-lightly.
-
-Because of the seriousness that language changes carry, any change must be
-beneficial to a large proportion of Python users. If the change only benefits a
-small percentage of Python developers then the change will not be made. A good
-way to see if your idea would work for a large portion of the Python community
-is to ask in the `Ideas Discourse category`_. You can also
-go through Python's stdlib and find examples of code which would benefit from
-your proposed change (which helps communicate the usefulness of your change to
-others). For further guidance, see :ref:`suggesting-changes`.
-
-Your proposed change also needs to be *Pythonic*. While only the Steering
-Council can truly classify something as Pythonic, you can read the
-:pep:`Zen of Python <20>` for guidance.
-
+need to be learned. Changes to Python are never taken lightly.
+
+Backward compatibility is a significant concern.  Existing Python code has to
+continue to work.  There are exceptions to this rule, but they are very rare
+and are only allowed when the benefits of the change greatly outweigh the costs
+of breaking existing code.
+
+Because of the seriousness of changing the language, any change must be
+beneficial to many Python users. If the change only helps a small percentage of
+Python developers then the change will not be made. A good way to see if your
+idea would work for a large portion of the Python community is to discuss it in
+the `Ideas category in Discourse <ideas_>`_.  You can also look in Python's
+standard library to find examples of code which would benefit from your
+proposed change.
+
+.. important::
+   For all of these reasons, most proposed changes to Python are rejected.  This
+   doesn't mean you shouldn't suggest them.  It can be useful to explore
+   alternatives and to get feedback from the community.  Just be aware that
+   getting a change accepted is difficult.  Don't take it personally.
```

Comment:
```suggestion
   For all of these reasons, most proposed changes to Python are rejected. This
   doesn't mean you shouldn't suggest them. It can be useful to explore
   alternatives and to get feedback from the community. Just be aware that
   getting a change accepted is difficult. Don't take it personally.
```

Reply:
I'll remove the double spaces......